task.sql

```diff
@@ -1,3 +1,13 @@
 USE ShopDB; 
 
--- Create your stored procedure here
+DELIMITER //
+CREATE PROCEDURE get_warehouse_product_inventory( 
+  IN WarehouseID INT
+)
+BEGIN
+  SELECT Products.Name AS ProductInventory, WarehouseAmount
+  FROM Products JOIN ProductInventory
+  ON Products.ID = ProductInventory.ProductID
+  WHERE ProductInventory.WarehouseID = WarehouseID;
+END //
+DELIMITER ;
```

Comment:
![image](https://github.com/user-attachments/assets/9dc881fc-1b20-4359-9c48-8c855b31ad10)
Add Eof(End-of-file), need to add a blank line at the end